host-bin/mount-chroot

```diff
@@ -237,8 +237,10 @@ $passphrase" | ecryptfs-wrap-passphrase "$wrappedfnek" -
             addtrap "rm -f '$wrappedkey' '$wrappedfnek'"
 
             # Extract wrapped keys from keyfile
-            tail -c 160 "$KEYFILE" | head -c 80 > "$wrappedkey"
-            tail -c 80 "$KEYFILE" > "$wrappedfnek"
+            wrappedtotal="`expr $(cat "$KEYFILE" | wc -c) - $(head -n 1 "$KEYFILE" | wc -c)`"
```

Comment:
`$((...))` is preferred over ``` `expr ...` ```
You can also skip the cat line without causing wc to output the file name by redirecting stdin.
Combined:
`"$(($(wc -c < "$KEYFILE") - ...))`"